target/linux/ramips/image/mt7621.mk

```diff
@@ -785,6 +785,17 @@ define Device/iptime_a3004t
 endef
 TARGET_DEVICES += iptime_a3004t
 
+define Device/iptime_a6004ns-m
+  $(Device/dsa-migration)
+  IMAGE_SIZE := 16128k
+  UIMAGE_NAME := a6004nsm
```

Comment:
Shouldn't it be `a6004nm`? How did you manage to install with this?

Reply:
There may be a problem because I wrote the file for personal use, but I rewritten it and uploaded it this time. The image name is a6004nm accurate, and we will revise it soon.

Reply:
I hope you've tested this at runtime, as I won't accept a PR that has not been tested properly.